exporter/pulsarexporter/README.md

```diff
@@ -0,0 +1,72 @@
+# Pulsar Exporter
+
+| Status                   |                       |
+| ------------------------ |-----------------------|
+| Stability                | [in-development]      |
```

Comment:
Please add the link at the bottom, see https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/c85ffc0da0463c37975fbad94dd3a002363d17a6/exporter/parquetexporter/README.md?plain=1#L32 as an example

Reply:
RESOLVED